src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -916,6 +921,48 @@ public void parse(InputFile inputFile, Format ext, int type, boolean thumbOnly,
 		}
 	}
 
+	/**
+	 * Extracts key value.
+	 *
+	 * @param key
+	 * @return If key is not available or blanc, NULL will be returned, otherwise string key value
+	 */
+	private String extractKeyValue(Tag t, FieldKey key) {
```

Comment:
For this method and `extractKeyIntegerValue`, can they have more descriptive names since they are for Tags? Something like `extractAudioTagKeyValue` and `extractAudioTagKeyIntegerValue`?

Reply:
Yes, I'll do that.